static/specification.html

```diff
@@ -93,34 +106,220 @@
+            &lt;link
+              rel="monetization"
+              href="https://example.com/pay"
+              onmonetization="sayThanks(this)"&gt;
+
+            &lt;script&gt;
+              function sayThanks(monetizedLink) {
+                // Do something here
+              }
+            &lt;/script&gt;
+          </pre>
+        </aside>
+      </section>
+      <section>
+        <h3>
+          Monetization events
+        </h3>
+        <p>
+          The {{MonetizationEvent}} DOM events provide information such as the
+          amount sent, the currency of the payment (what we call the "asset
+          code"), and a link to the incoming payment at the [=monetization
+          receiver=] which can be used to verify the receipt of payment. As in
+          the previous example above, these events are dispatched to [^link^]
+          elements and bubble up the DOM tree.
+        </p>
+        <p>
+          To listen for {{MonetizationEvent}} events, an [=event listener=]
+          needs to be added to the relevant [^link^] element (or to one of its
+          ancestors) via JavaScript:
+        </p>
+        <aside class="example">
+          <pre class="html">
+            &lt;link rel="monetization" href="https://example.com/pay"&gt;
+            &lt;script&gt;
+              const link = document.querySelector('link[rel="monetization"]');
+              link.addEventListener("monetization", event =&gt; {
+                // See how much was sent and in what currency
+                const { amountSent : { value, currency }} = event;
+                console.log(`Browser sent ${currency} ${value}.`);
+
+                // Use the incoming payment URL to verify the payment at the receiver.
+                verifyPayment(event);
+              });
+            &lt;/script&gt;
+          </pre>
+        </aside>
+      </section>
+      <section>
+        <h3>
+          Verifying a payment
+        </h3>
+        <p>
+          The {{MonetizationEvent}}'s {{MonetizationEvent/incomingPayment}}
+          attribute is a URL that can be used to verify the payment at the
+          [=monetization receiver=].
+        </p>
+        <p>
+          In most cases requests to the URL will need to be authenticated.
+          Below is a hypothetical verification endpoint that will make
+          authenticated requests to the receiver and return the results.
+        </p>
+        <aside class="example">
+          <pre class="JS">
+            /** @type {MonetizationEvent} event */
+            async function verifyPayment(event) {
+              // Legacy receivers don't support returning incoming payment URLs
+              if(!event.incomingPayment){
+                throw new Error('No incoming payment URL')
+              }
+
+              const body = JSON.stringify({
+                incomingPayment: event.incomingPayment
+              });
+
+              const response = await fetch('/verify', {
```

Comment:
I'm not sure if `/verify` is a good example since it's not part of the Open Payments spec.

Reply:
I have changed this example to simply query the incoming payment.